<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentInquiryWidget</class>
 <widget class="QWidget" name="PaymentInquiryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: rgb(98, 46, 255);
font: 22pt &quot;隶书&quot;;</string>
         </property>
         <property name="text">
          <string>业主缴费查询</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="widget">
        <attribute name="title">
         <string>已缴费</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(33, 111, 255);
font: 10pt &quot;隶书&quot;;</string>
               </property>
               <property name="text">
                <string>搜索</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_4">
               <property name="text">
                <string>缴费周期</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_5">
               <property name="text">
                <string>缴费时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_6">
               <property name="text">
                <string>缴费人姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="table2">
             <column>
              <property name="text">
               <string>订单编号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费周期</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费类型</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>应付</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>实付</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费时间</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费人用户名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费人姓名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>家庭地址</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>电话号码</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费方式</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(33, 111, 255);
font: 10pt &quot;隶书&quot;;</string>
               </property>
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget">
        <attribute name="title">
         <string>未缴费</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(33, 111, 255);
font: 10pt &quot;隶书&quot;;</string>
               </property>
               <property name="text">
                <string>搜索</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_1">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(33, 111, 255);
font: 10pt &quot;隶书&quot;;</string>
               </property>
               <property name="text">
                <string>发放账单</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="table">
             <column>
              <property name="text">
               <string>订单编号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费周期</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费类型</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>应付</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>实付</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费时间</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费人用户名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费人姓名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>家庭地址</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>电话号码</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>缴费方式</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(33, 111, 255);
font: 10pt &quot;隶书&quot;;</string>
               </property>
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
